vadim.martsun/task_6/src/main/java/com/hillel/spring/mvc/service/transactionExplorer/TransactionExplorerImpl.java

```diff
@@ -0,0 +1,30 @@
+package com.hillel.spring.mvc.service.transactionExplorer;
+
+import com.hillel.spring.mvc.model.Account;
+import com.hillel.spring.mvc.model.Transaction;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@Service
+public class TransactionExplorerImpl implements TransactionExplorer {
+    @Override
+    public void displayTransaction(Account account, Transaction transaction) {
+        String senderName = transaction.getSender().getUser().getFirstName()
+                + " "
+                + transaction.getSender().getUser().getLastName();
+
+        String receiverName = transaction.getReceiver().getUser().getFirstName()
+                + " "
+                + transaction.getReceiver().getUser().getLastName();
+
+        String status = (transaction.isSuccessful() ? "Successful" : "Failed");
+        if (transaction.getSender().equals(account)) {
+            log.info(senderName + " → " + receiverName + " ($" + transaction.getAmount() + ")"
```

Comment:
Don't use concatenation  in logging. Use {} 
Example of this logging: log.info("{} -> {} (${})", senderName, receiverName,  transaction.getAmount());

Reply:
OK :)